_projects/area_indicators.md

```diff
@@ -0,0 +1,60 @@
+---
+published: false # change to true once ready to post
+
+title: "Climate Change API Area Indicators" # project title inside quotes
+excerpt: "A high performance Scala API for calculating climate indicators over geospatial areas" # shows on project list page
+
+# project attributes
+requirements: # bullet list of requirements – one requirement per line – follow below format
+  - "Scala programming experience"
+  - "Python programming experience not required but preferred"
+  - "NetCDF experience not required but preferred"
+tags: # one tag per line – spaces are allowed in tags – follow below format
+  - "scala"
+  - "climate change"
+difficulty: "Hard" # easy, medium, hard – pick one
+mentors: # github username without the @ – example: designmatty
+  - "maurizi"
+  - "pcaisse"
+  - "jamesmcclain"
+  - "jisantuc"
+  - "fungjj92"
+
+# This file uses Kramdown. See https://kramdown.gettalong.org/syntax.html for syntax
+---
+
+The [Climate Change API](https://climate.azavea.com/) allows for simple location-based access to low temperature, high temperature and precipitation as well as 22 climate indicators derived from that data. It does this by preprocessing select parts of NetCDF data for individual locations, importing them into a SQL database, and performing queries on that data using an API written in Python. That approach has some limitations, so we built a proof-of-concept prototype. The prototype is written in Scala and uses [GeoTrellis](https://geotrellis.io) with a simpler ingest process to avoid needing a database, greatly reducing ingest and hosting costs, as well as allowing calculations on areas instead of just single points.
+
+This purpose of this project is to build on that existing proof-of-concept project to complete the features needed to bring it to parity with the existing API.
+
+## Milestones
+
+#### Preparation
+
+In the beginning of the project you will prepare by:
+
+- Familiarization with the Climate Change API
```

Comment:
hyperlink CCAPI here

Reply:
We link to the marketing site in the description, but I think this is a good spot to link to the docs